ilastik/workflows/pixelClassification/pixelClassificationWorkflow.py

```diff
@@ -303,7 +310,19 @@ def onProjectLoaded(self, projectManager):
         if self._headless:
             # In headless mode, let's see the messages from the training operator.
             logging.getLogger("lazyflow.operators.classifierOperators").setLevel(logging.DEBUG)
-        
+
+        if self.variable_importance_path: 
+            classifier_factory = self.pcApplet.topLevelOperator.opTrain.ClassifierFactory.value
+            classifier_factory.set_variable_importance_path( self.variable_importance_path )
+            
+        if self.trees_number:
+            classifier_factory = self.pcApplet.topLevelOperator.opTrain.ClassifierFactory.value
+            classifier_factory.set_num_trees( self.trees_number )
+            
+        if self.label_proportion:
+            classifier_factory = self.pcApplet.topLevelOperator.opTrain.ClassifierFactory.value
+            classifier_factory.set_label_proportion( self.label_proportion )
+            
         if self.retrain:
```

Comment:
If `--trees-number` or `--label-proportion`, you need to mark the classifier as being dirty.  Otherwise it won't be automatically be retrained (unless the user also provides the `--retrain` parameter.
`self.pcApplet.topLevelOperator.opTrain.ClassifierFactory.setDirty()`